### PR TITLE
Feature/message component

### PR DESCRIPTION
**Pull request by jtini**
## Description

Adds the styling and markup for the Message component to be used in the Feed and Messages.

## Type

- [X] Feature
- [ ] Bug
- [ ] Tech debt

### CODE REVIEW

File: packages/heartwood-components/components/message/message.hbs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<div class="message">
+    <span class="message__pre">
+        {{#if from.image}}
+        {{#if from.isSprucebot}}
+        <img class="message__from-image{{#if isSprucebot}} message__from-sprucebot-image{{/if}}" src="{{ from.image.src }}"
```
Comment (brandonminch) [inline packages/heartwood-components/components/message/message.hbs]: @jtini looks like the `message__from-sprucebot-image` class isn't being applied

File: packages/heartwood-components/components/04-images-and-icons/01-avatar/avatar.config.js [hunk 2]
```diff
@@ -15,35 +20,40 @@ module.exports = {
 			context: {
 				image
 			}
-		}, {
```
Comment (gcpantazis) [inline packages/heartwood-components/components/04-images-and-icons/01-avatar/avatar.config.js]: Just wanna be sure `prettier`/`eslint` is configured properly here, I thought our ruleset kept opening braces on the same line if chained (could be wrong about that though)

File: packages/heartwood-components/public/stylesheets/global.css [hunk 3]
```diff
@@ -4015,3 +4015,82 @@ select:-webkit-autofill:focus {
   @media screen and (min-width: 1000px) {
     .l-detail__content {
       padding: 0 2rem; } }
+
```
Comment (gcpantazis) [inline packages/heartwood-components/public/stylesheets/global.css]: Not to fix here, but we need to get build artifacts out of git history

File: packages/heartwood-components/components/message/message.hbs (hunk 1 above)
Comment (jtini) [reply packages/heartwood-components/components/message/message.hbs]: fixed

File: packages/heartwood-components/components/04-images-and-icons/01-avatar/avatar.config.js (hunk 2 above)
Comment (brandonminch) [reply packages/heartwood-components/components/04-images-and-icons/01-avatar/avatar.config.js]: @gcpantazis looks like the prettier settings are the same as other projects. The vscode settings were not part of the project though, I have added the settings for format on save to the other PR I opened.

**Pull request merged**